Doc/Tutorial/chapter_cookbook.tex

```diff
@@ -146,50 +146,51 @@ \subsection{Producing randomised genomes}
 Section~\ref{sec:Bio.SeqIO-and-StringIO}):
 
 \begin{minted}{python}
-import random
-from Bio.Seq import Seq
-from Bio.SeqRecord import SeqRecord
-from Bio import SeqIO
+>>> import random
```

Comment:
If switching to a doctest style, the minted format should be ``pycon``

However, unless this becomes an actively tested example, it is far more user-friendly to have it as a python formatter code block which they can copy and paste into their favourite editor.

Why change it?

Reply:
For consistency. Just imagine what Travis would say if he found out.

Reply:
I believe the long-term goal is to move the Biopython documentation to sphinx. Sphinx shows the python prompt by default but has the facility to hide it to simplify copying the code. If we are indeed moving to sphinx, then I can remove the prompts here for the current PR.

Reply:
That doesn't convince me. We have lots of long python snippets elsewhere in the Tutorial using ``\begin{minted}{python}`` and no ``>>>`` prefixes. It is infuriating as a user when copy-and-pasting a _long_ code snippet to have to remove the prefixes. There is no practical reason to use them here, the example has no output to show the user.

(And yes, some Python IDE tools are smart enough to remove the ``>>>`` for you, but they are a minority)

Reply:
That Sphinx magic sounds great. And yes, I do want to move the LaTeX to RST rendered along with the API using Sphinx, but haven't worked on this since a proof on principle at the BOSC 2019 CoFest last year.

Reply:
Thanks!